docs/rules/default-import-match-filename.md

```diff
@@ -0,0 +1,44 @@
+# import/default-import-match-filename
+
+Enforces default import name to match filename. Name matching is case-insensitive, and characters `._-` are stripped.
+
+## Rule Details
+
+### Options
+
+#### `ignorePaths`
+
+Set this option to `['some-dir/', 'bb']` to ignore import statements whose path contains either `some-dir/` or `bb` as a substring.
```

Comment:
should this support globs, and that should be in the example?

Reply:
It seems that globs is not a good tool against relative paths.

Reply:
can you elaborate? it seems like a good tool to me.

Reply:
To perform a glob matching a cwd is required, for example:
```js
// Using a hypothetical glob library
glob.match({
  pattern: 'tests/**/*.js'
  target: '/home/john/project-a/tests/a.js'
  cwd: '/home/john/project-a'
})
// true
```
What should the cwd be in this rule? The first thought is to use the location of `eslintrc` as cwd, but it is problematic when there is another `eslintrc` in a subdirectory, or when `overrides` is used.

Reply:
the cwd for any eslint rule is always `process.cwd()`, since you can't be sure where the config file is located.

Reply:
Using `process.cwd()` taken as the cwd means that the rule will give different result depending on the cwd of the shell, is that fine? 

```
// option for this rule
{
  ignoredPaths: ['tests/**/*.js']
}

project-root$ npx eslint . # will ignore files in tests/ 
project-root/tests$ npx eslint . # will **not** ignore files in tests/
```

Reply:
yes, that seems fine to me.